apcd-cms/src/apps/admin_regis_table/templates/list_registrations.html

```diff
@@ -77,6 +80,14 @@ <h1>List Registrations</h1>
       }
     })
   }
+  function openAction(reg_id) {
+    var dropdown, selectedOption, modal_id;
+    dropdown = document.getElementById('actionsDropdown');
+    selectedOption = dropdown.options[dropdown.selectedIndex].value;
+    modal_id = `${selectedOption}Modal_${reg_id}`;
+    $(`#${modal_id}`).modal({backdrop: "static"});
```

Comment:
<sup>**Reminder for Self**: Element is selected via Bootstrap out of convenience because jQuery is (probably) required for modal interaction.<sup>